controllers/droneController.js

```diff
@@ -0,0 +1,62 @@
+
+const Drone = require("./../models/Drone.model")
+
+exports.getAllDrones = async (req, res) => {
+    const allDrones = await Drone.find ({})
+    
+    res.render("drones/list", {
+        data: allDrones
+    })
+}
+
+exports.viewCreateDrone = async (req, res) => {
+    res.render("drones/create")
+}
+
+exports.createDrone = async (req, res) => {
+    const name = req.body.name
```

Comment:
req es un objeto accediento al body podemos destructurar las siguientes propiedades en una sola linea 😉 
```suggestion
    const {name, propellers, maxSpeed, image}  = req.body
```